migtests/tests/pg/datatypes/validate

```diff
@@ -46,9 +46,22 @@ EXPECTED_SUM_OF_COLUMN = {
 EXPECTED_ENUM_VALUES = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun']
 EXPECTED_BYTE_TYPE_VALUES = ['3031303130', '2d61626364', '34343538']
 
+#Due to the large sized source database requirements all tables will come up as colocated for now
+EXPECTED_COLOCATION_OF_TABLES = {
```

Comment:
This might not be useful as such in future. Since ideally we would want to fetch this list of colocated/sharded tables from the generated assessmentReport.json and use that list in our validations.

Reply:
We just need to know which table has which recommendation right? This won't change for every run?
I was thinking of adding a separate validation just for assessmentReport.json in the future.

Reply:
Nope this can change based on the any change in sizing algorithm in future. That change might not be very frequent. 
But the reports are the source of truth, and we should extract that from it.